### PR TITLE
ADR 977 - Invalid filename allowed upon upload

### PR DESCRIPTION
**Pull request by A-Souhei**
## Description
Added a environment variable in dev.env to be loaded by datapusher on start.

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [x] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [x] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [ ] New and existing tests pass locally with my changes.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -100,6 +100,8 @@ services:
       - ./submodules/datapusher:/var/www/datapusher
     ports:
       - "8800:8800"
+    environment:
+      DATAPUSHER_MAX_CONTENT_LENGTH: ${DATAPUSHER_MAX_CONTENT_LENGTH}
```
Comment (tomeksabala) [inline docker-compose.yml]: This doesn't look correct:
1. It's missing `-` at the start
2. It'd look better if you configure it to pass already existing env (just like with `CKAN_SITE_URL`)
```
environment:
  - DATAPUSHER_MAX_CONTENT_LENGTH
```
More info in docs: https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-environment-attribute

**Review by tomeksabala — CHANGES_REQUESTED** [review]
Could you update the PR title so it actually describes what change are you introducing? I get that it originated from work on the bugfix (leaving the Jira id is totally alright).
Just summarize the change as the wordy part of the title.

File: docker-compose.yml (hunk 1 above)
Comment (A-Souhei) [reply docker-compose.yml]: Thanks

File: docker-compose.yml [hunk 2]
```diff
@@ -104,6 +106,7 @@ services:
     tty: true
     command: /usr/local/bin/uwsgi --ini-paste deployment/datapusher-uwsgi.ini --honour-stdin
 
+
```
Comment (tomeksabala) [inline docker-compose.yml]: remove extra whitespace

File: docker-compose.yml [hunk 3]
```diff
@@ -96,6 +96,8 @@ services:
     image: adr.datapusher:${IMAGE_TAG}
     build:
       context: ./submodules/datapusher
+    environment:
+      - DATAPUSHER_MAX_CONTENT_LENGTH=${DATAPUSHER_MAX_CONTENT_LENGTH}
```
Comment (tomeksabala) [inline docker-compose.yml]: Writing here only:
```
- DATAPUSHER_MAX_CONTENT_LENGTH
```
will be enough as it tells docker-compose to pass the mentioned environment with the same value as exists on the host.

**Review by tomeksabala — CHANGES_REQUESTED** [review]

**Review by tomeksabala — APPROVED** [review]

**Comment by A-Souhei** [comment]
How can I merge this PR ? It looks like the test is failing and blocking the merge

**Pull request merged**